### ISSUE TITLE
Unable to add share

### ISSUE THREAD

**Issue by llamafilm** [cifsd-team/ksmbd-tools#272]
I installed ksmbd-tools version 3.4.4-1 on Ubuntu 22.04 by running `apt install ksmbd-tools` and I'm unable to add a new share.  Running as root:

```
# ksmbd.addshare --add-share=kmbd --options="
path = /mnt/raid    
read only = no
"
Usage: smbshareadd
	-a | --add-share=share
	-d | --del-share=share
	-u | --update-share=share
	-o | --options="op1=val1 op2=val2..."
	-c smb.conf
	-V | --version
	-v | --verbose
Supported share options:
	comment
	path
	guest ok
	guest account
	read only
	browseable
	write ok
	writeable
	store dos attributes
	oplocks
	create mask
	directory mask
	force create mode
	force directory mode
	force group
	force user
	hide dot files
	valid users
	invalid users
	read list
	write list
	admin users
	hosts allow
	hosts deny
	max connections
	veto files
	inherit owner
	follow symlinks
	vfs objects
	writable
```

If I try short-hand options the error is different:
```
# ksmbd.addshare -a ksmbd -o "path=/mnt/raid"
[ksmbd.addshare/30195]: ERROR: share name sanity check failure
```

I noticed this error in journalctl: `[ksmbd-worker/30338]: ERROR: Can't open '/etc/ksmbd/smb.conf': No such file or directory`

So I manually created that file and added my configuration to it.  But now I still get a similar issue trying to add a new user:
```
# ksmbd.adduser --add-user=MyUser
Usage: smbuseradd
	-a | --add-user=login
	-d | --del-user=login
	-u | --update-user=login
	-p | --password=pass
	-i smbpwd.db | --import-users=smbpwd.db
	-V | --version
```

And the journal shows this:
```
ksmbd: [ksmbd-worker/31121]: ERROR: Can't open `/etc/ksmbd/ksmbdpwd.db': No such file or directory
ksmbd: [ksmbd-worker/31121]: ERROR: User database file does not exist. Only guest sessions (if permitted) will work.
```

**Comment by llamafilm** [comment]
The short version of this command works: `ksmbd.adduser -a MyUser`.  After that the db error goes away.

**Comment by llamafilm** [comment]
Looks like this is already fixed in 3.4.5.
